language/evm/move-to-yul/src/generator.rs

```diff
@@ -90,7 +91,7 @@ impl Generator {
                 continue;
             }
             for fun in module.get_functions() {
-                if attributes::is_test_fun(&fun) {
+                if attributes::is_evm_test_fun(&fun) {
```

Comment:
Should this be for both test types?

Reply:
Not if we are going to remove `#[evm_test]`.